src/gaxios.ts

```diff
@@ -230,7 +231,7 @@ export class Gaxios {
   }
 
   private isReadableStream(obj: any): boolean {
```

Comment:
Could I trouble you to just drop this function and use the `isStream` method directly where applicable?

Reply:
Done